docs/content/topics/project-developers/events.md

```diff
@@ -9,4 +9,229 @@ aliases:
+context and information for utilization in handlers.
+
+The full list of fields on an Event is:
+
+  * [Source](#source)
+  * [Type](#type)
+  * [Payload](#payload)
+  * [ProjectID](#project-id)
+  * [Qualifiers](#qualifiers)
+  * [Labels](#labels)
+  * [Short Title](#short-title)
+  * [Long Title](#long-title)
+  * [Source State](#source-state)
+  * [Summary](#summary)
+
+Each field is reviewed in depth below.
+
+### Source
+
+An event's Source can be thought of as its domain. In fact, the rule of thumb
+to avoid clashes is to use a URI you control. This means leading with one's own
+domain or the URL for something else one owns, like the URL for a GitHub repo.
+As an example, events emitted from Brigade's [GitHub Gateway] use a source
+value of `brigade.sh/github`.
+
+### Type
+
+The Type of an event is used to specify increased granularity of the event's
+role/purpose as it relates to the source. For example, an event emitted by
+the GitHub Gateway with Type `push` signifies that a push event has occurred
+on a particular GitHub repo.
+
+### Payload
+
+An event Payload can be used to send free-form input that may be utilized when
+handling the event. One example would be sending the original GitHub push event
+payload on the corresponding GitHub gateway event, so that these additional
+details may be parsed in the Brigade script.
+
+### Project ID
+
+Although not normally used by general-purpose gateways, a Project ID value may
+be set on an event. In such cases, the event will _only_ be eligible for
+receipt by the project indicated by this fields' value.
+
+### Qualifiers
+
+For a project to receive a particular Eeent, the qualifiers on the project's
+Eeent subscription must exactly match the qualifiers on the event itself (in
+addition to matching Source and Type). For example, a project might supplement
+its `brigade.sh/github` event subscription with a qualifier of
+`repo: brigadecore/brigade`. Only events exactly matching this qualifier would
+be received by the project, i.e., only GitHub events on the brigadecore/brigade
+repository.
+
+### Labels
+
+Projects can choose to utilize Labels for filtering purposes. In contrast to
+qualifiers, the labels on a project's event subscription does not need to
+exactly match the labels on an Event in order to receive it. Labels, however,
+can be used to narrow an event subscription by optionally selecting only events
+that are labeled in a particular way. For example, a label of `branch: main`
+might be used on an event subscription to limit inbound GitHub events to only
+those that occur on the `main` branch of a given repo.
+
+### Short Title
+
+A short title may be provided for an event. This can then be utilized for
+logging, categorization or visual representation purposes, etc.
+
+### Long Title
+
+A longer, more descriptive title may be provided for an event. This may be
+helpful for providing additional context for users consuming event details.
```

Comment:
In v1, the only place we used them was Kashti. They got preserved in v2, but we haven't used them anywhere yet.

We don't have to get into the history, but do you think it's worth noting that we don't currently do anything with these fields and they're essentially reserved for future use?